addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -692,7 +692,8 @@ export function getDeepRange(editable, { range, sel, splitText, select, correctT
         correctTripleClick &&
         !endOffset &&
         (start !== end || startOffset !== endOffset) &&
-        (!beforeEnd || (beforeEnd.nodeType === Node.TEXT_NODE && !isVisibleStr(beforeEnd)))
+        (!beforeEnd || (beforeEnd.nodeType === Node.TEXT_NODE && !isVisibleStr(beforeEnd))) &&
+        !closestElement(previousLeaf(end), 'table')
```

Comment:
I understand we need to prevent triple click correction when the selection ends right after a table (which can't be a behavior of triple click), but better to write a comment to clarify it.
You should pass `editable` to `previousLeaf` (like we do just below) to make sure it doesn't go out of the editable. In fact it's probably best you just take line 698 out of the `if` and do `!closestElement(previous, 'table')` instead.